### PR TITLE
Fix test harness exit code on failure

### PR DESCRIPTION
**Pull request by ibgreen**
#### Background
For #1423
#### Change List
- Removes "circular" install of deck.gl, instead using alias to run from source, per discussion.
- Fixes passing of correct return status from Browser to Node.
- Also returns name of failing test.

### CODE REVIEW

File: src/test-utils/src/render-test.js [hunk 1]
```diff
@@ -122,11 +124,14 @@ export default class RenderTest {
 
   // Node test driver (puppeteer) may not have had time to expose the function
   // if the test suite is short. If not available, wait a second and try again
-  _reportToTestDriver(firstTime = false) {
+  _reportToTestDriver() {
     if (window.renderTestComplete) {
-      window.renderTestComplete(JSON.stringify(this.passed));
+      const result = {success: this.passed, failedTest: this.failedTest || ''};
```
Comment (jianhuang01) [inline src/test-utils/src/render-test.js]: understood we want to report the failed the test, but we won't stop on the failed test to give developer a chance to compare or save the new image.

Comment (ibgreen) [reply src/test-utils/src/render-test.js]: This line is intended to do that, let me test mode: https://github.com/uber/deck.gl/pull/1520/files#diff-e2d6c6241b75fd4782a84292b157689eR115

Comment (ibgreen) [reply src/test-utils/src/render-test.js]: I see, the browser script stops, but the test harness still kills the browser. Will fix.

Comment (ibgreen) [reply src/test-utils/src/render-test.js]: Fixed

**Pull request merged**